x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group1/create.ts

```diff
@@ -158,11 +159,6 @@ export default function createAlertTests({ getService }: FtrProviderContext) {
                   message: 'something important happened!',
                 },
```

Comment:
can we add a test that creating a normal (not system) action through the API without a `group` defined will throw an error?